azure-pipelines.yml

```diff
@@ -63,11 +57,14 @@ jobs:
       failTaskOnFailedTests: true
 
   - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@3
-    displayName: 'ESRP sign Microsoft.PowerPlatform.Formulas.Tools.dll'
+    displayName: 'Sign assemblies'
     inputs:
       ConnectedServiceName: 'ESRPCodeSigningConnection'
-      FolderPath: '$(Build.SourcesDirectory)/bin/$(BuildConfiguration)/PAModel/'
-      Pattern: '*.dll'
+      FolderPath: '$(Build.SourcesDirectory)/bin/$(BuildConfiguration)/'
+      Pattern: |
```

Comment:
just FYI, this one works too:
```
Pattern: 'PAModel/*.dll,Microsoft.PowerPlatform.PowerApps.Persistence/*.dll'
```

Reply:
yes, but I want to make sure we know exactly what we are signing